public/components/marketing-cards/30.html

```diff
@@ -0,0 +1,26 @@
+<div
+  class="flex flex-col overflow-hidden rounded-md selection:bg-blue-200 selection:text-blue-900"
+>
+  <div class="group relative">
+    <img
+      class="aspect-[16/9] w-full object-cover"
```

Comment:
I'm sure `aspect-video` is `16/9`.

Reply:
the `aspect ratio` plugin that you use overwrites the `aspect-video` class.
So, I'll implement it using `aspect-width-16` and `aspect-height-9` classes